cis/validation.py

```diff
@@ -0,0 +1,39 @@
+import json
+import os
+
+from jsonschema import validate as jsonschema_validate
+
+from cis.encryption import decrypt
+
+
+CIS_SCHEMA_JSON = os.path.join(os.path.abspath(os.path.dirname(__file__)), 'schema.json')
+
+
+with open(CIS_SCHEMA_JSON, 'r') as schema_data:
+    CIS_SCHEMA = json.load(schema_data)
+
+
+def validate(**payload):
+    """
+    Validates the payload passed to CIS.
+
+    :payload: Encrypted payload based on the output of `cis.encryption.encrypt` method
+    """
+
+    try:
+        # Decrypt payload coming from CIS using KMS key
+        # This ensures that publisher is trusted by CIS
+        decrypted_payload = decrypt(**payload)
+    except:
```

Comment:
Can we catch specific exceptions here so that we can avoiding silencing whatever may cause them? (I don't have a good idea atm)